arch/cortex-m0/src/lib.rs

```diff
@@ -166,34 +160,28 @@ core::arch::global_asm!(
 200: // MEXC_RETURN_MSP
   .word 0xFFFFFFF9
 300: // MEXC_RETURN_PSP
-  .word 0xFFFFFFFD"
-);
+  .word 0xFFFFFFFD",
```

Comment:
```suggestion
  .word 0xFFFFFFFD
  ",
```

This is more consistent with the style used in the other asm blocks, and hopefully avoids diff churn if we ever need to modify the end of the assembly block in the future.